.github/workflows/scheduled.yaml

```diff
@@ -39,3 +39,22 @@ jobs:
               "tag": "",
               "tag_latest": "true"
             }
+
+      - name: Log in to the Container registry 🗝
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@7b7aa264d83dc58691451798b4d117d53d21edfe
```

Comment:
```suggestion
        uses: aquasecurity/trivy-action@master
```

Reply:
Change to `master` everywhere to get the latest security patches, if and when Aqua makes them available

Reply:
done